### PR TITLE
stroopwafels!

### PR DESCRIPTION
**Pull request by Bierkraan**
## About the PR 
This PR adds stroopwafels that are made by putting 2 cookie dough and 1 maple syrup in an oven.

## Why's this needed? 
everone likes stroopwafels! (don't they?)

## Changelog

```
(u)Bierkraan:
(*)Added stroopwafels to the game!
```

### CODE REVIEW

File: code/datums/manufacturing.dm [hunk 1]
```diff
@@ -1935,8 +1935,9 @@ proc/get_nice_mat_name_for_manufacturers(mat)
 /datum/manufacture/hat_ltophat
 	name = "Large Top Hat"
 	item_paths = list("FAB-1")
+	item_names = list("Fabric")
 	item_amounts = list(5)
-	item_outputs = list(/obj/item/clothing/head/longtophat)
+	item_outputs = list(/obj/item/clothing/head/LongTophat)
```
Comment (pali6) [inline code/datums/manufacturing.dm:1940]: Why are you changing tophats?

File: code/obj/item/clothing/hats.dm [hunk 2]
```diff
@@ -435,7 +435,7 @@
 		..()
 		src.setMaterial(getMaterial("gold"))
 
-/obj/item/clothing/head/longtophat
+/obj/item/clothing/head/LongTophat
```
Comment (pali6) [inline code/obj/item/clothing/hats.dm:438]: Ditto.

File: config/admins.txt [hunk 3]
```diff
@@ -3,3 +3,8 @@
 #Most of the time, you'll want to make yourself a Host.
 #To do so, type your BYOND key with no spaces, capitals or other punctuation (including underscores!), a dash with spaces on both side, then the rank you want.
 #For example, the key "ZeWaka" would be given Host rank like this: zewaka - Host
+<<<<<<< HEAD
+=======
+
+bierkraan - Host
+>>>>>>> 00554f45c3470cfd70afb799cb9b1bbaff67ef8f
```
Comment (pali6) [inline config/admins.txt]:
```suggestion
```

**Review by pali6 — CHANGES_REQUESTED** [review]
Also please shorten the changelog to one entry. I don't think stroopwafels need 4 separata changelog sections.

File: code/modules/food_and_drink/snacks.dm [hunk 4]
```diff
@@ -190,6 +190,15 @@
 		..()
 		src.add_topping(0)
 
+/obj/item/reagent_containers/food/snacks/stroopwafel
+	name = "stroopwafel"
+	desc = "A traditional cookie from Holland, doesn't this need to go into the microwave?"
```
Comment (ZeWaka) [inline code/modules/food_and_drink/snacks.dm:195]:
```suggestion
	desc = "A traditional cookie from Holland. Doesn't this need to go into the microwave?"
```

**Review by ZeWaka — COMMENTED** [review]
rest looks fine

**Comment by Bierkraan** [comment]
can I add it to the wiki myself?

File: code/datums/manufacturing.dm [hunk 5]
```diff
@@ -1935,6 +1935,7 @@ proc/get_nice_mat_name_for_manufacturers(mat)
 /datum/manufacture/hat_ltophat
 	name = "Large Top Hat"
 	item_paths = list("FAB-1")
+	item_names = list("Fabric")
 	item_amounts = list(5)
 	item_outputs = list(/obj/item/clothing/head/longtophat)
 	time = 5 SECONDS
```
Comment (Adharainspace) [inline code/datums/manufacturing.dm:1941]: why is this here? not related to the pr at all

Comment (Bierkraan) [reply code/datums/manufacturing.dm:1941]: I have talked to Pali about this. I made an error writing in the wrong branch, but that's fixed

File: code/datums/manufacturing.dm [hunk 6]
```diff
@@ -1935,6 +1935,7 @@ proc/get_nice_mat_name_for_manufacturers(mat)
 /datum/manufacture/hat_ltophat
 	name = "Large Top Hat"
 	item_paths = list("FAB-1")
+	item_names = list("Fabric")
```
Comment (pali6) [inline code/datums/manufacturing.dm:1938]:
```suggestion
```
As I mentioned the Fabric item name description is unnecessary now as it's generated automatically.

**Review by pali6 — CHANGES_REQUESTED** [review]

**Review by pali6 — APPROVED** [review]
